ROADMAP.md

```diff
@@ -21,40 +27,34 @@ and interactive context, along with relationship discovery.
 
 - [x] Migration to the [Trait based API](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/doc/decisions/DR007-Hierarchical-or-compositional-traits-for-specifications.md).
 - [x] Migration to batch-centric API.
-- [ ] Migration of core API methods to C+. [In progress]
-- [ ] Sundry breaking changes and tech-debt cleanup.
+- [x] Migration of core API methods to C+.
```

Comment:
[fix] Not your change, typo.
```suggestion
- [x] Migration of core API methods to C++.
```

Reply:
Good spot thanks - fixed in the rebase [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/1116/commits/3f66154523d81c9cc9e80f048e5756ed1d753eb0#diff-683343bdf93f55ed3cada86151abb8051282e1936e58d4e0a04beca95dff6e51R30)